### PR TITLE
toolchain/gcc: Remove workaround for GCC 4.8

### PR DESCRIPTION
**Pull request by diizzyy**
GCC 4.8 was removed a long time ago...

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

File: toolchain/gcc/common.mk [hunk 1]
```diff
@@ -128,10 +128,6 @@ ifneq ($(CONFIG_mips)$(CONFIG_mipsel),)
   GCC_CONFIGURE += --with-mips-plt
 endif
 
-ifndef GCC_VERSION_4_8
```
Comment (wongsyrone) [inline toolchain/gcc/common.mk]: It means you should add the line below unconditionally instead of removing all of them.

Comment (diizzyy) [reply toolchain/gcc/common.mk]: Correct, managed to commit old patch. Thanks

File: toolchain/gcc/common.mk [hunk 2]
```diff
@@ -123,15 +123,12 @@ GCC_CONFIGURE:= \
 		--with-gmp=$(TOPDIR)/staging_dir/host \
 		--with-mpfr=$(TOPDIR)/staging_dir/host \
 		--with-mpc=$(TOPDIR)/staging_dir/host \
-		--disable-decimal-float
+		--disable-decimal-float \
+		--with-diagnostics-color=auto-if-env
```
Comment (ynezz) [inline toolchain/gcc/common.mk]: @diizzyy How is this related to this PR?

Comment (diizzyy) [reply toolchain/gcc/common.mk]: https://github.com/openwrt/openwrt/pull/1848/files#diff-2d94c69a28fa3c28c325f985e624a9d6L131 ?

**Comment by hauke** [comment]
Thank you for the patch, it was applied to master.
